src/main/java/org/openstreetmap/atlas/geography/atlas/dynamic/DynamicAtlas.java

```diff
@@ -162,6 +164,8 @@ public void buildUnderlyingMultiAtlas()
         {
             throw new CoreException("Cannot load shards with no data!");
         }
+        logger.trace("Built underlying MultiAtlas for {} in {}", this.getName(),
```

Comment:
It would be nice to keep the same logging pattern:
```
logger.trace("{}: Built underlying MultiAtlas in {}", this.getName(), buildTime.elapsedSince());
```

Reply:
Good point, let me update for consistency.